rollup.config.js

```diff
@@ -24,24 +24,28 @@ const LEGAL_TEXT = `/*
 * Attribution and license information can be found at https://github.com/MyPureCloud/client-app-sdk/blob/master/README.md
 */`;
 
+// Packages to exclude from esm/cjs bundles
+const EXTERNAL_DEPS = ['@babel/runtime', 'query-string'];
+
 const baseRollupConfig = {
     input: './src/index.js',
     cache: false,
     plugins: [
         commonjs(),
         resolve(),
-        json(),
-        babel({
-            exclude: 'node_modules/**'
-        })
+        json()
     ]
 };
 
-const buildRollupConfig = (config) => {
-    return Object.assign({}, baseRollupConfig, config);
-};
+const buildRollupConfig = (config) => Object.assign({}, baseRollupConfig, config);
+
+const umdBabelInputPlugin = getBabelInputPlugin({
+    babelHelpers: 'bundled',
```

Comment:
knowing what we now know about input and output rollup plugins, do you think we should have used transform-runtime in all cases and let rollup bundle up the dependencies based on the output format?  UMD would bundle it in vs cjs and esm would just leave the imports?

I'm not sure if this is possible, just curious now that we've learned more about all this.

Reply:
Even if that's possible but results in the helpers polluting the global namespace, I think this approach is better as we avoid pollution even if it's at the cost of our babel config complexity.

Reply:
Fiddled with this a bit tonight.  External seems to work with the external lib, but not with babel helpers.  They seem to get in-lined regardless of the runtime setting.

Reply:
Yeah, I think ideally we would be able to just use `babelHelpers: runtime` everywhere once the bug is fixed and rollup could probably manage to bundle up the imported babel helpers and I don't think this would pollute the global namespace. I think the issue you were running into was probably just the result of [this bug.](https://github.com/rollup/plugins/issues/475).